java/java.editor.base/test/unit/src/org/netbeans/modules/java/editor/base/semantic/DetectorTest.java

```diff
@@ -411,6 +441,19 @@ protected boolean process(CompilationInfo info, Document doc) {
         });
     }
     
```

Comment:
Jackpot:
warning: This anonymous inner class creation can be turned into a lambda expression.